pkg/kube/controllers/queue.go

```diff
@@ -135,7 +135,7 @@ var defaultSyncSignal = syncSignal{}
 // processed all events that were added prior to Run() being called Warning: these items will be
 // processed at least once, but may have failed.
 func (q Queue) HasSynced() bool {
-	return q.initialSync.Load()
+	return q.initialSync != nil && q.initialSync.Load()
```

Comment:
how does this get to be nil? seems NewQueue always set its

Reply:
This was leftover from a failed attempt to make TagWatcher or queues not leak goroutines if they are not run.  I will remove this.